client/landing/stepper/index.tsx

```diff
@@ -117,24 +90,27 @@ const FlowSwitch: React.FC< { user: UserStore.CurrentUser | undefined } > = ( {
 
 	user && receiveCurrentUser( user as UserStore.CurrentUser );
 
-	// console.log(flow);
-	// return;
 	return <FlowRenderer flow={ flow } />;
 };
 interface AppWindow extends Window {
 	BUILD_TARGET?: string;
 }
 
-declare const window: AppWindow;
-
 window.AppBoot = async () => {
+	// backward support the old Stepper URL structure (?flow=something)
+	const flowNameFromQueryParam = new URLSearchParams( window.location.search ).get( 'flow' );
```

Comment:
Could you clarify for my understanding when we have this scenario ? A flow name not being in the URL as /FlowName ?

Reply:
Before the work on making the url `/setup/link-in-bio`, it was `/setup?flow=link-in-bio`. 
So there could still be someone using the old link, therefore we're providing backward support for those.

Reply:
Thank you @escapemanuele, makes sense, good to keep track maybe we can remove it in the future ?

Reply:
Totally, I think it won't be long until the new version is the only one we need to support!